pyan/analyzer.py

```diff
@@ -1440,13 +1499,15 @@ def associate_node(self, graph_node, ast_node, filename=None):
     def add_defines_edge(self, from_node, to_node):
         """Add a defines edge in the graph between two nodes.
         N.B. This will mark both nodes as defined."""
-
+        status = False
         if from_node not in self.defines_edges:
             self.defines_edges[from_node] = set()
-        if to_node in self.defines_edges[from_node]:
-            return False
-        self.defines_edges[from_node].add(to_node)
+            status = True
         from_node.defined = True
+        if to_node is None or to_node in self.defines_edges[from_node]:
+            status = status or False
```

Comment:
Line 1508 does nothing, regardless of the old value of `status`?

Reply:
indeed. deleted the line